src/hooks/transfers/skipClient.tsx

```diff
@@ -63,7 +57,17 @@ const useSkipClientContext = () => {
             throw new Error(`Error: no rpc endpoint found for chainId: ${chainId}`);
           },
         },
-        getEVMSigner: async () => getEVMSigner(walletClientRef.current, sourceAccount),
+        getEVMSigner: async (chainId: string) => {
```

Comment:
do we need getCosmosSigner as well in case the route includes osmosis/noble?

Reply:
it does not need the signer if the initial transaction does not originate from osmosis or noble. i will add it soon though for the keplr/solana deposit flows